bootstrap/index.js

```diff
@@ -17,20 +17,32 @@ const withStatusCode = (execute) => (data, callback) => {
 const cacheOptions = envOptions()
 if (cacheOptions.enabled) logger.info('Cache enabled: ', cacheOptions)
 
+const withMiddleware = async (execute) => {
+  return await withCache(withStatusCode(execute))
+}
```

Comment:
Unneeded async usage
```suggestion
const withMiddleware = (execute) => withCache(withStatusCode(execute))
```